projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java

```diff
@@ -83,94 +91,41 @@ public static void saveCalendar(Path csvPath, String path) {
-      for (CSVRecord record : parser) {
-        if (isHeader) {
-          isHeader = false;
-          continue;
+        if (isServiceIdRelevantToday(record)) {
+          Protos.Calendar.Builder calendar = Protos.Calendar.newBuilder();
+          
+          calendars.add(calendar
+              .setServiceId(Integer.parseInt(record.get("service_id")))
+              .setSunday(Boolean.parseBoolean(record.get("sunday")))
+              .setMonday(Boolean.parseBoolean(record.get("monday")))
+              .setTuesday(Boolean.parseBoolean(record.get("tuesday")))
+              .setWednesday(Boolean.parseBoolean(record.get("wednesday")))
+              .setThursday(Boolean.parseBoolean(record.get("thursday")))
+              .setFriday(Boolean.parseBoolean(record.get("friday")))
+              .setSaturday(Boolean.parseBoolean(record.get("saturday")))
+              .setStartDate(Integer.parseInt(record.get("start_date")))
+              .setEndDate(Integer.parseInt(record.get("end_date")))
+              .build());
         }
-        FareAttributes.Builder fareAttributes = FareAttributes.newBuilder();
-        fareAttributes
-            .setFareId(Integer.parseInt(record.get("fare_id")))
-            .setPrice(Double.parseDouble(record.get("price")))
-            .setCurrencyType(record.get("currency_type"))
-            .setPaymentMethod(Integer.parseInt(record.get("payment_method")))
-            .setTransfers(Integer.parseInt(record.get("transfers")))
-            .setAgencyId(Integer.parseInt(record.get("agency_id")))
-            .setTransferDuration(record.get("transfer_duration"))
-            .build()
-            .writeDelimitedTo(output);
       }
+      return calendars;
     } catch (IOException e) {
       throw new RuntimeException(e);
     }
   }
 
-  public static void saveFareRules(Path csvPath, String path) {
-    try (FileOutputStream output = new FileOutputStream(path)) {
-      CSVParser parser =
-          CSVParser.parse(
-              csvPath,
-              StandardCharsets.UTF_8,
-              CSVFormat.DEFAULT.withHeader(
-                  "fare_id", "route_id", "origin_id", "destination_id", "contains_id"));
-      boolean isHeader = true;
-      for (CSVRecord record : parser) {
-        if (isHeader) {
-          isHeader = false;
-          continue;
-        }
-        FareRules.Builder fareRule = FareRules.newBuilder();
-        fareRule
-            .setFareId(Integer.parseInt(record.get("fare_id")))
-            .setRouteId(Integer.parseInt(record.get("route_id")))
-            .setOriginId(Integer.parseInt(record.get("origin_id")))
-            .setDestinationId(Integer.parseInt(record.get("destination_id")))
-            .setContainsId(record.get("contains_id"))
-            .build()
-            .writeDelimitedTo(output);
-      }
-    } catch (IOException e) {
-      throw new RuntimeException(e);
+  public static List<Integer> getRoutesId(){
+    List<Integer> routesId = new ArrayList<>();
```

Comment:
You can use Java 8 streams here to simplify:
`import java.util.stream.Collectors;`
Then,
`return getRoutes(routesTxt).stream().map(Route::getRouteId).collect(Collectors.toList());`